### PR TITLE
Removing K_POINTS card from CP

### PR DESCRIPTION
**Pull request by degennar**
When passing `kpts = (0,0,0)` then no `K_POINTS` card is inserted in the input file.
Please don't be too disgusted... :)

### CODE REVIEW

**Comment by elinscott** [comment]
Hmmm, I can see why you went for this solution, but I'm not 100% convinced `kpoints = (0,0,0)` is a natural solution. How about not printing whenever `kpoints = "exclude"`?

File: ase/io/espresso_cp.py [hunk 1]
```diff
@@ -73,6 +73,7 @@
 def write_espresso_cp_in(fd, atoms, input_data=None, pseudopotentials=None,
```
Comment (elinscott) [inline ase/io/espresso_cp.py:73]: and then I would change this to not take the arguments `kspacing` and `kpts` and `koffset`

**Comment by elinscott** [comment]
What do you reckon?

File: ase/io/espresso_cp.py (hunk 1 above)
Comment (degennar) [reply ase/io/espresso_cp.py:73]: Sounds good!

**Comment by degennar** [comment]
I agree with everything!
Actually I thought of doing something like `kpts="exclude"` but I was not sure about having `kpts` as a string also. But if this is fine for you I also think it's much better.
I will commit all your suggestions now.

**Comment by elinscott** [comment]
Yeah I know it's not that great having `kpts` as a string too, but python allows this sort of thing and I imagine it would be possible to  inadvertently set `kpts = (0, 0, 0)` somewhere else in the code without intending to drop off the `kpoints` block

**Review by elinscott — APPROVED** [review]
Looks great!

**Pull request merged**